lib/metasploit/framework/common_engine.rb

```diff
@@ -33,12 +33,14 @@ module Metasploit::Framework::CommonEngine
     end
 
     config.root = Msf::Config::install_root
+    config.paths.add 'app/models', autoload: true
```

Comment:
Since `application_record.rb` is excluded from the gem build this should be fine without a guard on how the code launched.

Reply:
Since `application_record.rb` needs to be restored in packaging, this may need a guard to determine if the path should be added at launch.